src/app/message/list-entry/message-list-entry.component.styl

```diff
@@ -29,7 +29,7 @@ flag
   border: 3px solid transparent
 
 p[md-line]
-  white-space: normal
+  white-space: normal !important
```

Comment:
what kind of selector do you mean? it gets always overriden when I try a different one.